cloud-template/build.gradle

```diff
@@ -18,4 +18,8 @@ dependencies {
 
     compile group: 'org.slf4j',                     name: 'slf4j-api',                      version: slf4jApiVersion
 
+    testCompile        group: 'org.springframework.boot',  name: 'spring-boot-starter-test',       version: springBootVersion
```

Comment:
Why is the `spring-boot-starter-test` needed for this module?

Reply:
added during  new tests ended up not being needed.